docroot/profiles/custom/cgov_site/themes/custom/cgov/src/libraries/analytics/AppMeasurement.custom.js

```diff
@@ -27,6 +27,8 @@ var AppMeasurementCustom = {
     /** Valid Cgov host identifiers. */
     cgovHosts: [
         // 'www.devbox',
+        // 'www.devbox',
```

Comment:
Why is the commented out line in twice?

Reply:
Because evidently I copied the line to modify the copy and then just added a new line instead.